Applications/Resnet/PyTorch/main.py

```diff
@@ -0,0 +1,143 @@
+# SPDX-License-Identifier: Apache-2.0
+# Copyright (C) 2023 DongHak Park <donghak.park@samsung.com>
+#
+# @file   main.cpp
+# @date   26 Jan 2023
+# @see    https://github.com/nnstreamer/nntrainer
+# @author Donghak Park <donghak.park@samsung.com>
+# @bug	  No known bugs except for NYI items
+# @brief  This is Resnet Example for PyTorch (only training)
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+
+from torch.utils.data import DataLoader
+from torchvision import datasets
+from torchvision.transforms import ToTensor, transforms
+
+DEVICE = "cpu"
+print(f"Using {DEVICE} device")
+print(f"PyTossrch version: {torch.__version__}")
```

Comment:
There is a typo error in version message of pytorch.

Reply:
[Fixed] Thank you !!!